<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageResizerDialogClass</class>
 <widget class="QDialog" name="ImageResizerDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Resizer</string>
  </property>
  <widget class="QPushButton" name="browseSourceFileButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseMediaDirectoryButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mediaDirectoryEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="reportEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceFileEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sourceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source file</string>
   </property>
  </widget>
  <widget class="QLabel" name="mediaLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Media directory</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
